package.json

```diff
@@ -3,14 +3,16 @@
   "version": "2.0.0",
   "description": "Simple module to calculate mandates using the D'Hondt method.",
   "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
```

Comment:
Also, I have no `index.d.ts` file!

Reply:
@jesusgn90 hi! `index.d.ts` is also auto-generated. It wasn't generated before because `"declaration": true` was missing in `tsconfig.json`.  `index.d.ts` is created by `tsc` based on the interfaces you already defined in your code. It should be bundled in the NPM package so that Typescript users can take advantage of those interfaces.